components/Customer/CustomerCard/index.js

```diff
@@ -0,0 +1,25 @@
+import CustomerForm from "../CustomerForm";
+import styled from "styled-components";
+
+const CustomerSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  border: 3px solid black;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  margin-bottom:20px;
+  
+`;
+const H1 = styled.h1`
```

Comment:
I suggest a name which reflects the styled component: `StyledHeadlineOne` or similar.